apps/home/templates/apps_home/blocks/faq_block.html

```diff
@@ -3,35 +3,39 @@
 {% get_identifier value as identifier %}
 <div class="row mb-4">
     <div class="col-md-8 offset-md-2">
-        <div class="accordion">
-            <a
-                id="accordion-{{ identifier }}-title"
-                href="#accordion-{{ identifier }}-body"
-                class="accordion__title collapsed"
-                aria-haspopup="true"
-                aria-expanded="false"
-                data-toggle="collapse">
-                <h2>
-                    {{ value.title }}
-                    <i class="fa fa-chevron-down" aria-hidden="true"></i>
-                </h2>
-            </a>
+        {% if self.background_color == "yellow" %}
+        <div class="accordion yellow">
+            {% elif self.background_color == "pink"%}
+            <div class="accordion pink">
+                {% elif self.background_color == "purple"%}
+                <div class="accordion purple">
+                    {% else %}
+                    <div class="accordion">
+                        {% endif %}
```

Comment:
The indentation for the ifs until the else should be the same. And is the colour choice required? In that case you do not need an `else` or can use it for your favourite colour.

Reply:
Ah, you use it for white? So, when it is not set it's white? I think that info could go into a help text for the colour field.

Reply:
Hmm, it's not working to not set a colour with a dropdown, right? So you need to add the white as option.

Reply:
If I add white as an Option, then I need another migration. Right now the buttons are "Pink", "Purple", "Yellow", and "----".

Reply:
I changed it, but I left the else for white (it's not my favorite, but I figured as it's the default, it makes the most sense) :)